MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+recursive-include aiofiles *.py
```

Comment:
The line should be removed: setuptools/distutils has a rule to include all `.py` files from mentioned packages.

Reply:
This is true.